src/less/variables/base/layout-variables.less

```diff
@@ -0,0 +1,10 @@
+@spacing-extra-small: 8px;
+@spacing-small: 16px;
+@spacing-medium: 24px;
+@spacing-large: 32px;
+@spacing-extra-large: 64px;
+
+@page-grid-cols-small: 8;
+@page-grid-cols-large: 16;
```

Comment:
I wonder if it would be good to have a `page-grid-cols-medium`?

Reply:
We only have two grids. 8 or 16.